docs/.vitepress/config/shared.ts

```diff
@@ -0,0 +1,113 @@
+import { defineConfig } from 'vitepress'
+
+// TODO:
+// export const META_IMAGE = 'https://test-utils.vuejs.org/social.png'
+export const META_IMAGE = null
+export const isProduction =
+  process.env.NETLIFY && process.env.CONTEXT === 'production'
+
+if (process.env.NETLIFY) {
+  console.log('Netlify build', process.env.CONTEXT)
+}
+
+const rControl = /[\u0000-\u001f]/g
+const rSpecial = /[\s~`!@#$%^&*()\-_+=[\]{}|\\;:"'“”‘’<>,.?/]+/g
+const rCombining = /[\u0300-\u036F]/g
+
+/**
+ * Default slugification function
+ */
+export const slugify = (str: string): string =>
+  str
+    .normalize('NFKD')
+    // Remove accents
+    .replace(rCombining, '')
+    // Remove control characters
+    .replace(rControl, '')
+    // Replace special characters
+    .replace(rSpecial, '-')
+    // ensure it doesn't start with a number
+    .replace(/^(\d)/, '_$1')
+
+export const sharedConfig = defineConfig({
+  title: 'Vue Test Utils',
+  appearance: 'dark',
```

Comment:
Please default to light mode

Reply:
@lmiller1990 In this new version of vitepress it is only possible to set `true`, `'dark'`, and `false`, so I set it to `true` for the theme to be based on the user preference.  `false` stop the user to be able to toggle between the themes.